.changeset/eight-ghosts-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@flatfile/plugin-webhook-forward-events': patch
```

Comment:
For our plugins, we typically start with a category in the name. Current categories are: extractors, transform, automation, export, schemas, datasets, migrations, core. Does this feel like it fits in any of these categories? I can't decide.

Reply:
@flatfile/plugin-automation-webhook-event-forwarding?